src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -38,6 +38,18 @@ public class ApplicationInsightsLoggerOptions : IOptionsFormatter
         /// </summary>
         public bool EnablePerformanceCountersCollection { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the flag that enables quick pulse collection.
+        /// Enabled by default.
+        /// </summary>
+        public bool EnableQuickPulse { get; set; } = true;
```

Comment:
Suggest to use the name LiveMetrics as thats the name of public feature. (QuickPulse was an internal name)
https://docs.microsoft.com/en-us/azure/azure-monitor/app/live-stream